src/ui/src/partials/request_view.html

```diff
@@ -95,9 +95,8 @@ <h1 class="page-header">
       <div class="col-md-6">
         <div style="margin-bottom: 10px;">
           <span class="h3">Output
-            <button type="button" class="btn pull-right" ng-click="download();">
-              <i class="fa fa-download" ></i> Download
-            </button>
+              <a class="fa fa-download pull-right" ng-href="{{downloadHref}}" download="{{filename}}"
+                 style="visibility:{{downloadVisible}};"></a>
```

Comment:
Angular has a special directive for this, `ng-show` - let's use that instead of setting the style.